src/nspawn/nspawn.c

```diff
@@ -1530,21 +1532,23 @@ static int resolved_listening(void) {
 
         r = sd_bus_open_system(&bus);
         if (r < 0)
-                return r;
+                return log_debug_errno(r, "Failed to open system bus: %m");
 
         r = bus_name_has_owner(bus, "org.freedesktop.resolve1", NULL);
-        if (r <= 0)
-                return r;
+        if (r < 0)
+                return log_debug_errno(r, "Failed to check whether resolved bus name is taken: %m");
```

Comment:
I think it'd be nicer to just say "Failed to check if org.freedesktop.resolve1 name is taken: %m". This requires less context to understand the log line.